native/acter/src/api/news.rs

```diff
@@ -386,54 +387,55 @@ pub struct NewsEntryDraft {
 
 impl NewsEntryDraft {
     pub fn add_text_slide(&mut self, body: String) -> &mut Self {
+        let text_content = TextMessageEventContent::markdown(body);
         self.slides.push(NewsSlide {
             client: self.client.clone(),
             room: self.room.clone().into(),
-            inner: news::NewsSlide::new_text(body),
+            inner: news::NewsSlide {
+                content: NewsContent::Text(text_content),
+                references: Default::default(),
+            },
```

Comment:
I prefer it the way it was before. I explain more in the comment around the section change.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/978/commits/ef091de0739d8d31a253cec981d154fb4d4895ec